src/components/Button.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import styled from '@emotion/styled';
 import PropTypes from 'prop-types';
 
-export default function Button({ children, color, onClick }) {
+export default function Button({ type, children, color, onClick }) {
```

Comment:
```suggestion
export default function Button({ children, ...props }) {
```
이렇게 하고(그럼 prop에 type, color, onClick 및 그 외 추가될 속성들이 들어있는 상황)